doc/docs/revision_history.md

```diff
@@ -2,6 +2,7 @@
 
 | Date       | Description                                                                                                                                                                                  |
 |------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| November 2025 | - Updated the [Supported operating systems](./os_support.md) page: Windows 11 is now Tier 1, Windows 10 is now Tier 3, macOS 15 is now Tier 2, and macOS 13 is now Tier 3.                |
```

Comment:
We can update macOS 15 to Tier 1: https://nordicsemi.atlassian.net/browse/NRFU-1715?focusedCommentId=2182917